.github/workflows/e2e.yaml

```diff
@@ -0,0 +1,52 @@
+name: Cypress E2E tests
+
+on:
+  push:
+    branches: [main]
```

Comment:
Ideally, for all our CI checks we'd have some logic here:
if (merged via PR) {
    do not trigger
} else {
    assume it's a direct push to main and trigger
}

I think I've wasted at least an hour or two trying to figure out GitHub even logic to make it work.

However, we don't actually have visibility into any checks that run outside of a PR — there are no notifications. Unless we were to set up something like a Slack channel for failure alerts, my vote at this time is to remove this trigger on push to main.